dw-al2-hbase-ami/dw-hbase-ami-install.sh

```diff
@@ -0,0 +1 @@
+../dw-al2-emr-ami/dw-emr-ami-install.sh
```

Comment:
Symlink

Reply:
just create a new one.  We don't know if this will change in the future.

Reply:
Duplicate code until this changes in the future though?

Reply:
for now.  It should be treated as a separate piece of work.